pkg/controller/registry/reconciler/reconciler.go

```diff
@@ -192,6 +201,9 @@ func Pod(source *operatorsv1alpha1.CatalogSource, name, opmImg, utilImage, img,
 				"kubernetes.io/os": "linux",
 			},
 			ServiceAccountName: saName,
+			// If this field is not set, there is a chance that the pod is admitted to the cluster by the admission webhook without
+			// the imagePullSecrets defined for it's serviceAccount.
```

Comment:
In the race, the service account does not have secrets yet - you are guaranteed to get them if they exist.

Reply:
How about:
```
/// If this field is not set, there that the imagePullSecrets defined in the pod
// and its serviceAccount will be out of sync.
```

Reply:
sure, something like that works